art/attacks/__init__.py

```diff
@@ -21,6 +21,10 @@
 from art.attacks.evasion.virtual_adversarial import VirtualAdversarialMethod
 from art.attacks.evasion.zoo import ZooAttack
 
+# Added New Attacks
```

Comment:
Please remove this comment line and the empty line 23
```suggestion
```

Reply:
Removed.

Reply:
Removed Unwanted lines.